content/en/sensitive_data_scanner/_index.md

```diff
@@ -57,7 +57,7 @@ There are two locations where you can redact your sensitive data:
 **In your environment:**
 
 {{< callout url="https://www.datadoghq.com/private-beta/sensitive-data-scanner-using-agent-in-your-premises/" >}}
-  Sensitive Data Scanner using the Agent is in private beta. To request access, fill out this form.
+  Sensitive Data Scanner support for the Datadog Agent is in beta. To enroll, click <strong>Request Access</strong>.
```

Comment:
Acesss forms and the default header still say `beta`, so am leaving as beta for now.